x/fees/spec/02_state.md

```diff
@@ -8,27 +8,27 @@ order: 2
 
 The `x/fees` module keeps the following objects in state:
 
-| State Object        | Description                 | Key                                    | Value                        | Store |
-| :------------------ | :-------------------------- | :------------------------------------- | :--------------------------- | :---- |
-| `DeployerAddress`   | Deployer address bytecode   | `[]byte{1} + []byte(contract_address)` | `[]byte{deployer_address}`   | KV    |
-| `WithdrawAddress`   | Withdraw address bytecode   | `[]byte{2} + []byte(contract_address)` | `[]byte{withdraw_address}`   | KV    |
-| `ContractAddresses` | Contract addresses bytecode | `[]byte{3} + []byte(deployer_address)` | `[]byte{contract_addresses}` | KV    |
+| State Object     | Description                           | Key                                                               | Value         | Store |
```

Comment:
@MalteHerrmann The deployed version renders fine for me on the current specs:
<img width="1904" alt="Screen Shot 2022-07-04 at 15 45 01" src="https://user-images.githubusercontent.com/33006856/177167509-be91fc1a-5b61-405f-96f2-c51a5fdbdef3.png">
 
I tried to run this locally with `make docs-serve`, but after building unfortunately I can only see a white screen locally. 

Could you add a screenshot of what is not rendering correctly?

Reply:
You can see on your screenshot that `DeployerAddress` and other things are broken into two lines. When flipped, it might fit into one line as it should. Just a cosmetic, really.